core/object/script_language.cpp

```diff
@@ -102,6 +102,22 @@ Dictionary Script::_get_script_constant_map() {
 	return ret;
 }
 
+int Script::get_script_method_argument_count(const StringName &p_method, bool *r_is_valid) const {
```

Comment:
Fallback in the same way, this means that as long as an extended script has `get_method_info` with appropriate return information you can utilize the whole chain of functionality, so this should work out of the box for existing extensions that provide method information